### PR TITLE
Clarify ftSignatureItems for Germany, include details about QR code

### PR DESCRIPTION
**Pull request by TSchmiedlechner**

### CODE REVIEW

File: doc/appendix-de-kassensichv/reference-tables/type-of-signature-ftsignaturetype.md [hunk 1]
```diff
@@ -4,37 +4,44 @@ title: 'Type of signature: ftSignatureType'
 ---
 
 # Type of Signature: ftSignatureType
+The `ftSignatureType` indicates the type and origin of the signature. The data type is `Int64` and can contain a country-specific code (following the ISO-3166-1-ALPHA-2 standard), converted from ASCII into hex and used as byte 8 and 7 (e.g. `4445` for `DE`).
```
Comment (volllly) [inline doc/appendix-de-kassensichv/reference-tables/type-of-signature-ftsignaturetype.md:7]:
```suggestion
The `ftSignatureType` indicates the type and origin of the signature. The data type is `Int64` and can contain a country-specific code (following the ISO-3166-1-ALPHA-2 standard), converted from ASCII into hex and used as byte 8 and 7 (e.g. `0x4445` for `DE`).
```

File: doc/appendix-de-kassensichv/reference-tables/type-of-signature-ftsignaturetype.md [hunk 2]
```diff
@@ -4,37 +4,44 @@ title: 'Type of signature: ftSignatureType'
...
+| `0x444500000000001D` | Signature                                               | Only when using **text** | 1.3                    |
+| `0x444500000000001E` | Public key of the TSE                                   | Only when using **text** | 1.3                    |
+| `0x444500000000001F` | Process start (first action)                            | Yes*                     | 1.3                    |
+| `0x4445000000000020` | Update-transaction-payload                              | No                       | 1.3.14                 |
+| `0x4445000000000021` | Update-transaction-result                               | No                       | 1.3.16                 |
+| `0x4445000000000022` | Certification identification (e.g. BSI-K-TR-1234-5678). Also contains information about the certification state (for non certified TSEs) and the state of the environmental protection. | Yes                      | 1.3.16                 |
+
+
+\* This value must be printed when using long-running business sequences and/or the implicit flow and **is not included in the QR code**. It is computed with the following formula: `min(cbReceiptMoment, min(chargeItems.Moment), min(payItems.Moment))`
```
Comment (volllly) [inline doc/appendix-de-kassensichv/reference-tables/type-of-signature-ftsignaturetype.md:42]:
```suggestion
\* This value must be printed when using long-running business sequences and/or the implicit flow and **must not be included in the QR code**. It is computed with the following formula: `min(cbReceiptMoment, min(chargeItems.Moment), min(payItems.Moment))`
```

Comment (TSchmiedlechner) [reply doc/appendix-de-kassensichv/reference-tables/type-of-signature-ftsignaturetype.md:42]: Actually it is correct as it is 😅 thanks tough

File: doc/appendix-de-kassensichv/reference-tables/type-of-signature-ftsignaturetype.md [hunk 3]
```diff
@@ -4,37 +4,43 @@ title: 'Type of signature: ftSignatureType'
...
+| `0x4445000000000012` | _Finish transaction_ result                             | Only when using **text** | 1.3                    |
+| `0x4445000000000013` | QR code version                                         | No                       | 1.3                    |
+| `0x4445000000000014` | POS serial number                                       | Only when using **text** | 1.3                    |
+| `0x4445000000000015` | TSE process type                                        | Only when using **text** | 1.3                    |
+| `0x4445000000000016` | TSE process data                                        | Only when using **text** | 1.3                    |
+| `0x4445000000000017` | Transaction number                                      | Only when using **text** | 1.3                    |
+| `0x4445000000000018` | Signature counter                                       | Only when using **text** | 1.3                    |
+| `0x4445000000000019` | Start time of the _start transaction_                   | No                       | 1.3                    |
```
Comment (AxelKutschera) [inline doc/appendix-de-kassensichv/reference-tables/type-of-signature-ftsignaturetype.md:30]:
```suggestion
| `0x4445000000000019` | Log time of the _start transaction_                   | No                       | 1.3                    |
```
- Only a suggestion, maybe I´m wrong.

File: doc/appendix-de-kassensichv/reference-tables/type-of-signature-ftsignaturetype.md [hunk 4]
```diff
@@ -4,37 +4,43 @@ title: 'Type of signature: ftSignatureType'
...
+| `0x444500000000001A` | Log time of the _finish transaction_                    | Only when using **text** | 1.3                    |
+| `0x444500000000001B` | Signature algorithm                                     | Only when using **text** | 1.3                    |
+| `0x444500000000001C` | Log time format                                         | Only when using **text** | 1.3                    |
+| `0x444500000000001D` | Signature                                               | Only when using **text** | 1.3                    |
+| `0x444500000000001E` | Public key of the TSE                                   | Only when using **text** | 1.3                    |
+| `0x444500000000001F` | Process start (first action)                            | Yes*                     | 1.3                    |
```
Comment (AxelKutschera) [inline doc/appendix-de-kassensichv/reference-tables/type-of-signature-ftsignaturetype.md:36]: - Should be added, that all the "times" have to be printed in UTC?

**Comment by AxelKutschera** [comment]
According to KassenSichV § 6 2. the "Datum der Belegausstellung" (and 3. and 5.), some more information has to be printed. Should this be added here?

**Review by AxelKutschera — COMMENTED** [review]
Some suggestions.

**Pull request closed without merging**